Microsoft.Azure.CosmosRepository/src/Dtos/ILimitedResultRequest.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace Microsoft.Azure.CosmosRepository.Dtos
```

Comment:
```suggestion
﻿namespace Microsoft.Azure.CosmosRepository.Pagination
```

Need the copyright header.